### ISSUE TITLE
flake8 checks in pre-commit fail

### ISSUE THREAD

**Issue by fridex** [thoth-station/adviser#2335]
## Bug description

```
tests/prescription/v1/test_group.py:37:28: F541 f-string is missing placeholders
tests/prescription/v1/test_group.py:185:28: F541 f-string is missing placeholders
tests/sieves/test_experimental_conf_index.py:150:21: F541 f-string is missing placeholders
tests/sieves/test_experimental_conf_index.py:168:21: F541 f-string is missing placeholders
thoth/adviser/resolver.py:1372:21: F541 f-string is missing placeholders
thoth/adviser/boots/environment_info.py:[92](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/thoth-station_adviser/2334/pre-commit/1524015462336172032#1:build-log.txt%3A92):32: F541 f-string is missing placeholders
thoth/adviser/boots/environment_info.py:99:32: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:219:28: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:316:28: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:416:28: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:516:28: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:617:28: F541 f-string is missing placeholders
tests/prescription/v1/test_add_package_step.py:719:28: F541 f-string is missing placeholders
tests/prescription/v1/test_gh_release_notes.py:311:28: F541 f-string is missing placeholders
thoth/adviser/prescription/v1/unit.py:5[99](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/thoth-station_adviser/2334/pre-commit/1524015462336172032#1:build-log.txt%3A99):25: F541 f-string is missing placeholders
thoth/adviser/boots/thoth_search.py:62:28: F541 f-string is missing placeholders
```

**Comment by VannTen** [comment]
Is this still relevant ? does not seem to fail when I run it locally

**Comment by VannTen** [comment]
Fixed in #2341
/close
